topics/visualisation/tutorials/bad-graphs/slides.html

```diff
@@ -0,0 +1,285 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+license: MIT
+title: Friends Don't Let Friends Make Bad Graphs
+contributions:
+  authorship:
+  - cxli233
+  editing:
+  - hexylena
+---
```

Comment:
shall we add the `video: true` tag here? Seems like a good candidate for an automated video :)